sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -340,11 +340,11 @@ Create a configuration client with proxy options.
 <!-- embedme ./src/samples/java/com/azure/data/appconfiguration/ReadmeSamples.java#L175-L187 -->
 ```Java
 // Proxy options
-final String HOST_NAME = "{your-host-name}";
```

Comment:
Is this fixed by @conniey PR already?

Reply:
Ah, yes, I see this now, I reviewed that PR before this change was added.

Reply:
Buahaha, guess who gets to fix this merge conflict.

Reply:
What I'm surprised by.. is that the app configuration pipeline didn't catch this.

Reply:
That is being looked into since it is a severe issue.